### PR TITLE
Fix multiple patches not working correctly

### PR DESCRIPTION
**Pull request by mwm5945**
### :pencil: Description
- Fixes the sample charts to pull from correct repository
- updates the echo server to use the alpine tag to get the example working on newer M1 Macs
- Fixes the path being appended multiple times for the JSON patch when multiple items are specified for the sidecar. 

### :link: Related Issues
- #43 
- #42

### CODE REVIEW

**Comment by mwm5945** [comment]
@mvaalexp when you get a chance, are you able to take a look at this? :D

File: charts/kubernetes-sidecar-injector/values.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ container:
 
 podAnnotations: {}
 podLabels: {}
+securityContext: {}
```
Comment (mvaalexp) [inline charts/kubernetes-sidecar-injector/values.yaml:17]: can we please change this to 
`podSecurityContext`

**Review by mvaalexp — APPROVED** [review]

**Review by mvaalexp — APPROVED** [review]

**Comment by mvaalexp** [comment]
Ok, apologies. Looks like this is do to this (i.e., GH secrets dont wonk with forks).
https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow

Sorry, give me a bit to fix this. When I inherited this it was originally using travis and I didn't catch it on my branches.

**Comment by mvaalexp** [comment]
Going to go ahead and merge this as my other approvers are busy and this passed unit tests (which is as far as my changes are handling).
https://github.com/ExpediaGroup/kubernetes-sidecar-injector/pull/46

**Comment by mvaalexp** [comment]
Expected, this is only a docker change, chart didn't change (unless i missed it)

**Comment by mvaalexp** [comment]
Oh.. the podSecurityContext.... sorry!
Can you bump the Chart version (minor revision).

**Pull request merged**